.github/ISSUE_TEMPLATE/help_desk.md

```diff
@@ -0,0 +1,41 @@
+---
+name: Help Desk
+about: Despite multiple attempts you repeatedly fail to solve an issue?  
+title: "[Help Desk]"
+labels: help desk
+assignees: ''
+
+---
+ 
+## Please note
+<!-- A brief description of what you expected to happen. -->
+- We may ignore any issues due to a lack of time and apologize.
+- Please first always refer to the [documentation](https://k-wave-python.readthedocs.io/en/latest/index.html), or specific [examples](https://github.com/waltsims/k-wave-python/tree/master/examples) before raising an issue
+- Code dumps "can you help, here is my code" will be closed without comment 
+- Be as specific as possible and write in detail what you have already tried, add links and any reference that may help us understand 
```

Comment:
- Add any links and references to resources you have referenced in your problem solving.